test/parsing/king-11/CLEANFILES

```diff
@@ -0,0 +1,3 @@
+aaaaaa*
+longName.good
+longNameHelper*
```

Comment:
There's no need to move these files unless you'd like to clean up a bit, but increasingly we're trying to sort files into directories by meaning rather than by username (but not worrying about cleaning up the past).  So I think that `test/parsing/errors/` would be a better place for this test.

Reply:
moved the directory into `test/parsing/errors`